src/contentScripts/index.ts

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable no-console */
 import { onMessage } from 'webext-bridge'
 import { createApp } from 'vue'
 import App from './views/App.vue'
+import plugin from '~/logic/plugin'
```

Comment:
```suggestion
import contextPlugin from '~/logic/context-plugin'
```

Reply:
Sorry, I added your suggestion too fast.

I'm not sure to rename this file from `plugin` to `context-plugin` is a good idea in fact.
As you can see in [`plugin.ts`](https://github.com/antfu/vitesse-webext/pull/109/files#diff-12ac833824d101aa66950a3877dd7e4590d65c0b6f9cf000f860e3e277d4e0dcR8-R12), it's inject `$app` and provide an object with `context`. So it's a bit confusing and provide future conflict.

I see these options :
- We change for `app.config.globalProperties.$context` and `app.provide('context', ...` instead of `$app`. So we can rename the file `context-plugin.ts` and use directly `$context` in vue template like [here in the `SharedSubtitle.vue`](https://github.com/antfu/vitesse-webext/pull/109/files#diff-136b285d3bc00b14ab8ce4dc655d4e3d155e954d7fb7510362cabc737b284adaR3).
- We use more explicitly the file `plugin.ts` as a base for developers to add their business logic injected in `globalProperties` with `$app` and `app` in script setup. Or to add future features. We can maybe rename `plugin.ts` to `app.ts` or `global-app.ts`, or `injected.ts`, or other. I don't know.

This second option involves using `$app.context` in Vue template like `<p>This is the {{ $app.context }} page</p>` but provide destructuring in script setup like this :
```vue
<script setup lang="ts">
const { context } = inject('app')
console.log(context)
</script>
```
Or maybe `app.provide('app', app.config.globalProperties.$app)` is too much. I don't know what is the best to improve DX...

Reply:
We can also remove this commit that add `plugin.ts` and the `$app` `globalProperties` to simply use `import { getCurrentContext } from 'webext-bridge'` in the `SharedSubtitle.vue` component.
It's maybe too much and out of the scope of this branch.
But it will be interesting to get a global plugin dedicated for web extension and vuejs, and discuss that feature in a dedicated issue...

Reply:
Ok, then do you think it could be better to name it as `common-setup`, where you pass the app reference, instead of being a plugin?

```ts
import { setupApp } from './logic/common-setup'

const app = createApp()
setupApp(app)
app.mount('#app')
```

This way, ppl could install additional plugins in one place for both contexts.

Reply:
Excellent!
I pushed changes and added examples in `common-setup` file.